src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // перетворити рядок на масив де елемент це правило css
+  // перевірити чи рядок не порожній
+
+  const listStyles = sourceString
+    .split(';')
+    .filter((string) => string.trim() !== '');
+
+  const stylesObject = {};
+
+  listStyles.forEach((entry) => {
+    const [key, value] = entry.split(':').map((el) => el.trim());
+
+    if (key && value) {
```

Comment:
The conditional check ensures that only valid key-value pairs are added to the object, which is a good practice.